src/case-swapping.js

```diff
@@ -16,6 +16,13 @@
  */
 function swapCase(phrase) {
   // write code here
+  return phrase.split('').map(function(item) {
```

Comment:
```suggestion
  return phrase.split('').map((item) =>  {
```
arrow function would be better